### ISSUE TITLE
[a11y] Record set visual loading issue

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1129]
**Describe the bug**
Record set is loading in an erratic and disruptive way. This does not meet WCAG 2.1 standards due to fast, disruptive movement.

**To Reproduce**
1. Click on **Record Sets** in the nav bar
2. See issue

**Expected behavior**
![sp7-edge](https://user-images.githubusercontent.com/37256050/146943793-a055c151-3811-42bc-81b7-18c23e935e57.gif)

**Actual behavior**
![load in sequence](https://user-images.githubusercontent.com/37256050/146943820-7c4502db-7213-4dab-b654-39d07112bf1e.gif)

**Desktop (please complete the following information):**
 - OS: Windows 10 and macOS Monterey
 - Browser: Edge & Safari
